contracts/mechanics/garden-of-forking-paths/GardenOfForkingPaths.sol

```diff
@@ -63,7 +76,7 @@ library LibGOFP {
         uint256 numSessions;
         mapping(uint256 => Session) sessionById;
         // session => stage => stageReward
-        mapping(uint256 => mapping(uint256 => StageReward)) sessionStageReward;
+        mapping(uint256 => mapping(uint256 => Reward)) sessionStageReward;
```

Comment:
I don't think this will be a problem, but we need to check if this change will corrupt data when we upgrade existing contracts.

The reason I think it won't is because of how struct storage positions are calculated: https://docs.soliditylang.org/en/v0.8.20/internals/layout_in_storage.html